### DISCUSSION TITLE
how report event with Manual on nodejs sdk

### DISCUSSION THREAD

**Discussion by dengshenkk** [apache/skywalking#9399] [Q&A] [answered]
There is only one way to start on the document.

**Comment by wu-sheng** [comment, marked as answer]
I don't think we have manual SDK on the nodejs agent, right? @kezhenxu94

**Reply by kezhenxu94** [reply]
> I don't think we have manual SDK on the nodejs agent, right? @kezhenxu94

Hi, we don't have this in nodejs agent for now

**Reply by dengshenkk** [reply]
我的意思是在某些时机手动上报一些自定义的数据

I mean to manually report some custom data at some point.

**Reply by dengshenkk** [reply]
我在别的地方看到类似这样的代码, 但是没有看到对应的文档
I saw code like this in other places, but I didn't see the corresponding document

```javascript
const { ContextManager } = require('skywalking-backend-js');

const span = ContextManager.currentSpan
span.tag({
    key:"req",
    val:['requestData']
});
span.tag({
    key:"res",
    val:['responseData']
});
span.async();
```
